Backend/router/v1/report.ctrl.js

```diff
@@ -20,32 +20,32 @@ const showReportList = (req, res) => {
 }
 
 const readReport = (req, res) => {
-  const report_ID = parseInt(req.params.report_ID)
+  const reportId = parseInt(req.params.reportId)
   //When ID is not a number
-  if (Number.isNaN(report_ID)) {
+  if (Number.isNaN(reportId)) {
     return res.status(400).end()
   }
   models.report.findAll({
     where: {
-      id: report_ID
+      id: reportId
     }
   }).then((result) => {
     res.json(result);
   }).catch((err) => {});
 }
 
 const destroyReport = (req, res) => {
```

Comment:
destroy is a strong word :) if the functionality is deleting a report, just deleteReport should be good enough